octokit/issues.go

```diff
@@ -1,49 +1,80 @@
 package octokit
 
 import (
-	"net/url"
-	"time"
-
 	"github.com/jingweno/go-sawyer/hypermedia"
+	"time"
 )
 
 // RepoIssuesURL is a template for accessing issues in a particular
 // repository for a particular owner that can be expanded to a full address.
-var RepoIssuesURL = Hyperlink("repos/{owner}/{repo}/issues{/number}")
+var RepoIssuesURL = Hyperlink("repos/{owner}/{repo}/issues{/number}?" +
+	"filter={filter}&state={state}&labels={labels}&sort={sort}&" +
```

Comment:
I think this should be:

```go
Hyperlink("repos/{owner}/{repo}/issues{/number}{?filter,state,labels,sort}")
```

[RFC 6750][rfc] makes for some great bedtime reading. :night_with_stars: 

[rfc]: https://tools.ietf.org/html/rfc6570